src/plugins/visualize/public/application/utils/get_table_columns.tsx

```diff
@@ -167,12 +175,14 @@ export const getNoItemsMessage = (createItem: () => void) => (
   <EuiEmptyPrompt
     iconType="visualizeApp"
     title={
-      <h1 id="visualizeListingHeading">
-        <FormattedMessage
-          id="visualize.listing.createNew.title"
-          defaultMessage="Create your first visualization"
-        />
-      </h1>
+      <EuiTitle size="l">
+        <h1 id="visualizeListingHeading">
+          <FormattedMessage
+            id="visualize.listing.createNew.title"
+            defaultMessage="Create your first visualization"
```

Comment:
Let's leave this as is because its in the visualization

Reply:
this is the page content header for create visualization. Given the requirement, I think we should wrap it with `EuiText `
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/32652829/e9125235-f6f5-4341-8b7c-a77c1672097c)